### ISSUE TITLE
Gesture: swipe up anywhere now shows Menu 

### ISSUE THREAD

**Issue by joechengla** [mozilla-tw/Zerda#1013]
swipe up near the bottom of the screen makes sense but i can also trigger the menu by swiping up at at the top of the screen which seems weird. UX?

**Comment by dakaizou** [comment]
To be more specific, anywhere means anywhere on the homescreen. Doing this in settings and browsing pages doesn't trigger the menu.

**Comment by joechengla** [comment]
right. only on the browser homescreen. you can swipe up near the notification bar and menu will show

**Comment by dakaizou** [comment]
Swiping down on the homescreen will trigger search bar
